browser.js

```diff
@@ -307,9 +307,11 @@ function noop () {}
 /* eslint-disable */
 /* istanbul ignore next */
 function pfGlobalThisOrFallback () {
-  const defd = (o) => typeof o !== 'undefined' && o
+  var defd = function (o) {
```

Comment:
may as well just do `function defd` and remove the var assign

Reply:
Done.